react-basic-practice/src/components/CustomerForm/index.tsx

```diff
@@ -112,6 +112,11 @@ const CustomerForm: React.FC<CustomerFormProps> = ({
               name="status"
               defaultValue={data?.status || "Open"}
               onBlur={handleBlur}
+              _focusVisible={{}}
```

Comment:
why need to use empty object

Reply:
because I need to override chakraUI's default property